src/Action.hs

```diff
@@ -133,6 +134,14 @@ class MonadError ActionExcept m => ActionError m
 
 -- * Action Combinators
 
+loginByUser :: ActionUserHandler m => User -> m ()
+loginByUser = login . view _Id
```

Comment:
The issue I have with such functions is that this easy to end-up looking a user by id in the DB just to call this function which then discards all but the Id.